read-fonts/src/tables/postscript/encoding.rs

```diff
@@ -0,0 +1,22 @@
+//! CFF standard encoding.
+
+/// See "Standard" encoding at <https://adobe-type-tools.github.io/font-tech-notes/pdfs/5176.CFF.pdf#page=37>
```

Comment:
Despite the comment I have no idea what these magical bytes mean if I just glance at this. The referenced content does not appear to explain what a cff standard endocing is. Let's add a few more words explaining what this is.

Reply:
Added a brief explanation and a link to the page in the spec where this _is_ actually discussed